src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
It's not a scalable solution to manually add each day of the calendar. Consider using a scripting language like JavaScript to dynamically generate the calendar days. This will allow for easier maintenance and adjustments for different month lengths.